File: Slip 22/Core Java/program1.java
import java.util.Scanner;

class program1 {
    int factorial(int n) {
        if (n <= 1) {
            return 1; 
        }
        else {
            return n * factorial(n - 1);
        }
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int num = sc.nextInt();
        sc.close();
        program1 obj = new program1();
        System.out.println("The factorial of " + num + " is " + obj.factorial(num));      
    }
}

File: Slip 24/Python/program1.py
def prime(no):
    if no < 2 or no % 2 == 0:
        return False
    for i in range(3, int(no**0.5) + 1, 2):
        if no % i == 0:
            return False
    return True
def factorial(n):
    sum = 1
    for i in range(1, n + 1):
        sum *= i
    return sum
number = int(input("Enter a number: "))
if prime(number):
    print(number, "is a prime number.")
else:
    print(number, "is not a prime number.")
print("The factorial of", number, "is", factorial(number))
